src/types/onyx/PersonalDetails.ts

```diff
@@ -0,0 +1,45 @@
+type PersonalDetails = {
+    /** ID of the current user from their personal details */
+    accountID: number;
+
+    /** First name of the current user from their personal details */
+    firstName?: string;
+
+    /** Last name of the current user from their personal details */
+    lastName?: string;
+
+    /** Display name of the current user from their personal details */
+    displayName?: string;
```

Comment:
This is also required - AFAIK there's no case we should NOT have a user's display name

```suggestion
    displayName: string;
```

Reply:
Adjusted, thanks 👌